facia/app/controllers/FaciaController.scala

```diff
@@ -218,13 +218,13 @@ trait FaciaController extends BaseController with Logging with ImplicitControlle
 
   private def getPressedCollection(collectionId: String): Future[Option[PressedCollection]] =
```

Comment:
Are we sure this function is not used by endpoints that requires the full pressed file? Same question below for `getSomeCollections`?

Reply:
I'm not certain. We need to list the endpoints require the full PressedPage.

For this PR, I've assumed that only `/*path/show-more/*id.json`and `/*path/deduped.json` require the full version.

Reply:
let's try to figure it out. Let me know if you need any help.